src/test/common/errorHandling.ts

```diff
@@ -25,18 +25,21 @@ describe('ErrorHandling', () => {
 
   it('should invoke given request handler', async () => {
     const handled = ErrorHandling.apply(resolvingRequestHandler)
+    // @ts-ignore
```

Comment:
why are these being added? would it not be better to fix the lint issue rather than ignore it?